packages/env/lib/config/validate-config.js

```diff
@@ -10,6 +10,25 @@
  */
 class ValidationError extends Error {}
 
+/**
+ * Validates that the value is a string.
+ *
+ * @param {string} configFile The configuration file we're validating.
+ * @param {string} configKey  The configuration key we're validating.
+ * @param {number} value      The value to check.
+ */
+function checkString( configFile, configKey, value ) {
+	if ( value === undefined ) {
```

Comment:
Why not throw if undefined? Wouldn't we only call `checkString` once we know an option was set?

Reply:
It _does_ look like none of the validations are called when they might be `undefined`. I can probably remove this case from all of the `checkXXX` functions entirely.